core/pom.xml

```diff
@@ -51,6 +51,13 @@
             <artifactId>org.eclipse.emf.ecore.xmi</artifactId>
         </dependency>
 
+        <!-- InfluxDB -->
+        <dependency>
+            <groupId>com.influxdb</groupId>
+            <artifactId>influxdb-client-java</artifactId>
+            <version>1.7.0</version>
+        </dependency>
```

Comment:
Version number should be a maven property, and the full dependency should be in the _dependencyManagement_ tag instead of the _dependencies_ one (see how it's done for the other dependencies).
I'll take care of it.